internal/exec/options.go

```diff
@@ -0,0 +1,76 @@
+package exec
+
+import (
+	"syscall"
+
+	"github.com/Microsoft/hcsshim/internal/jobobject"
+	"golang.org/x/sys/windows"
+)
+
+type ExecOpts func(e *execConfig) error
+
+type execConfig struct {
+	dir                   string
+	env                   []string
+	stdout, stderr, stdin bool
+
+	job      *jobobject.JobObject
+	procAttr *syscall.SysProcAttr
```

Comment:
This is probably useless, can just take out the fields we'll actually get use of.